app/config/sculpin_kernel.yml

```diff
@@ -26,3 +26,8 @@ sculpin_content_types:
             - category
             - authors
         permalink: pretty
+    8:
```

Comment:
@bmackinney creating a content type seems to be the only way to generate a list type page for a dir, I tried a few different ways but this was my only working state as of yesterday. Open to suggestions or alternatives. 

Long term I think we could use different types to generate dynamic content for trending docs related to the user's current page, plus what you mentioned above.